### ISSUE TITLE
Poor portable instance support.

### ISSUE THREAD

**Issue by therealcharmics** [ModOrganizer2/modorganizer#1920]
It used to be, the first time you launched MO2, you'd be presented with an option to use it portably. This has always been my preferred way as it avoids cross-directory dependencies and allows me to keep my game management clean and reliable. However, this option no longer exists, and I am instead forced to create an instance upon first launch of MO2. 

Latest version
Windows 10

**Comment by Silarn** [comment]
You mean like this?

![image](https://github.com/ModOrganizer2/modorganizer/assets/826174/7b7782ae-fba1-4a69-b3b7-dd5ab0823ec2)

**Comment by Silarn** [comment]
With no global instances created, this is the second page of the 'Creating a new instance' dialog that opens when you first start MO2.

It is the same dialog that displays when you hit 'create new instance' in the instance manager.

**Comment by therealcharmics** [comment]
Thank you for the swift reply! That behavior is exactly what I expected, so I've tested on another system and haven't been able to reproduce the issue. On the first PC, its skips right past that window, going straight to instance configuration. Unfortunately, I haven't been able to get logs either, since this is part of the first time set-up.

Regardless, it seems to be an issue unique to my computer. Closing as "can't reproduce". I'll re-open if I'm able to get anything useful.
